pkg/asset/cluster/tfvars.go

```diff
@@ -434,6 +434,10 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 				return err
 			}
 			createFirewallRules = permissions.Has(GCPFirewallPermission)
+
+			if !createFirewallRules {
+				logrus.Warnf("failed to find permission %s, skipping firewall rule creation", GCPFirewallPermission)
```

Comment:
Can GCPFirewallPermission be "" ?

Reply:
`GCPFirewallPermission` is a constant set here
https://github.com/openshift/installer/blob/186cb916c388d29ed3f6ef4e71d9fda409f30bdf/pkg/asset/cluster/tfvars.go#L79